### PR TITLE
View collection exercise state on ce details page

### PR DESCRIPTION
**Pull request by gemmai95**
Dependent on US011. Covers the UI for US047, US042, US040, US028 and US041. Adds the collection exercise state to the collection exercise details page.

### CODE REVIEW

**Comment by gemmai95** [comment]
[US047: Create and View 'Live' State of a CE [M]](https://trello.com/c/6htalBsL/37-us047-create-and-view-live-state-of-a-ce-m)

File: response_operations_ui/templates/collection-exercise.html [hunk 1]
```diff
@@ -14,6 +14,14 @@ <h1 class="jupiter" name="page-ce-title">{{survey.shortName}} {{ce.exerciseRef}}
         <li id="sample-success" class="system-feedback__item success">Sample successfully loaded</li>
       {% endif %}
     </ul>
+    <dl class="ce-info">
+        <dt class="ce-status__title ce-status">
+          Status:
+        </dt>
+        <dd class="ce-status__data ce-status">
+          ce.state
```
Comment (jcox-dev) [inline response_operations_ui/templates/collection-exercise.html]: Missing curly brackets for template variable.

**Comment by emilioward** [comment]
Isn't this PR the same as Faisal's, whats happening to that PR now?

**Comment by gemmai95** [comment]
I am going to have a word with Mark about that

**Comment by gemmai95** [comment]
[US011 Display State of a Collection Exercise [M]](https://trello.com/c/kJO9QkLd/29-us011-display-state-of-a-collection-exercise-m)

**Comment by jcox-dev** [comment]
[US040: Create and View 'Created' State of a CE [M]](https://trello.com/c/TI7VB3Jr/34-us040-create-and-view-created-state-of-a-ce-m)

**Pull request merged**